generate.py

```diff
@@ -0,0 +1,116 @@
+from pathlib import Path
+
+from aind_data_schema_models.generators._base import (
+    _HarpDeviceTypeModel,
+    _ModalityModel,
+    _PlatformModel,
+    _SpeciesModel,
+    _MouseAnatomyModel,
+    _RegistryModel,
+)
+from aind_data_schema_models.generators.generators import (
+    GeneratorContext,
+    MappableReferenceField,
+    ModelGenerator,
+    ForwardClassReference,
+)
+from aind_data_schema_models.generators.parsers import csv_parser, get_who_am_i_list
+
+if __name__ == "__main__":
+
+    root = Path(__file__).parent / r"src/aind_data_schema_models/models"
+    target_folder = Path(r".\src\aind_data_schema_models\_generated")
```

Comment:
We could probably have this use a similar way that root is defined?

Reply:
We could put the generator code in a separate repo and bundle it into a docker image (or pypi package). We could then just put the instructions in the readme like: 
`docker run -e {parameters specifying the input and output directories} name-of-docker-image:latest`